R/trackhub.R

```diff
@@ -0,0 +1,69 @@
+### =========================================================================
+### TrackHub support
+### -------------------------------------------------------------------------
+
+### - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+### TrackHub class
+###
+
+setClass("TrackHub", representation(uri = "character"))
+
+isFileReference <- function(s) {
+  length(grep(".txt", s)) == 1
```

Comment:
What about just `tools::file_ext(s) == "txt"`?

Reply:
Yeah, this looks good